.ci_support/migrations/cuda_112_ppc64le_aarch64.yaml

```diff
@@ -0,0 +1,46 @@
+migrator_ts: 1645421738
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+  paused: false
+  override_cbc_keys:
+    - cuda_compiler_stub
+  operation: key_add
+  check_solvable: false
+  primary_key: cuda_compiler_version
+  wait_for_migrators:
+    - aarch64 and ppc64le addition
+  exclude_pinned_pkgs: False
+  use_local: true
+  commit_message: "Rebuild for cuda for ppc64le and aarch64"
+
+arm_variant_type:              # [aarch64]
+  - sbsa                       # [aarch64]
+
+c_compiler_version:            # [ppc64le or aarch64]
+  - 10                         # [ppc64le or aarch64]
+cxx_compiler_version:          # [ppc64le or aarch64]
+  - 10                         # [ppc64le or aarch64]
+fortran_compiler_version:      # [ppc64le or aarch64]
+  - 10                         # [ppc64le or aarch64]
+
+cuda_compiler_version:         # [ppc64le or aarch64]
+  - 11.2                       # [ppc64le or aarch64]
+
+cudnn:                  # [ppc64le or aarch64]
+  - 8                   # [ppc64le or aarch64]
+
+cdt_name:  # [ppc64le or aarch64]
+  - cos7   # [ppc64le or aarch64]
+
+docker_image:                                           # [os.environ.get("BUILD_PLATFORM", "").startswith("linux") and (ppc64le or aarch64)]
+   # case: native compilation (build == target)
+   - quay.io/condaforge/linux-anvil-ppc64le-cuda:11.2   # [ppc64le and os.environ.get("BUILD_PLATFORM") == "linux-ppc64le"]
+   - quay.io/condaforge/linux-anvil-aarch64-cuda:11.2   # [aarch64 and os.environ.get("BUILD_PLATFORM") == "linux-aarch64"]
+   # case: cross-compilation (build != target)
+   - quay.io/condaforge/linux-anvil-cuda:11.2           # [ppc64le and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+   - quay.io/condaforge/linux-anvil-cuda:11.2           # [aarch64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
```

Comment:
Not sure if adding this migration is strictly speaking necessary, but since the linux builds have both cuda / non-cuda, I think we should do the same for aarch/ppc?

This reflects the changes in https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3624, which we should ideally merge before-hand (then we can skip the `use_local: true` here).

Reply:
No problem, I'll remove it; though I wonder if we'll still be able to cross-compile cuda without this (which is the point of this PR, no?)

Reply:
How do you propose we solve the part in https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/210#issuecomment-1412993900?

Reply:
> How do you propose we solve the issue in [#210 (comment)](https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/210#issuecomment-1412993900)?

I don't know yet, I'm trying to help work through the issues as they appear. But the overarching goal remains to cross-compile CUDA, so if we're falling short of that, we should keep this open IMO.